spec/features/user/user_can_have_role_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
```

Comment:
@jmejia: should this be a model test? I'm thinking that the fact that a user has a role is not a functionality.